test-suite/src/test/scala/org/scalajs/testsuite/javalib/util/ObjectsTest.scala

```diff
@@ -0,0 +1,70 @@
+package org.scalajs.testsuite.javalib.util
+
+import java.{util => ju}
+
+import org.scalajs.jasminetest.JasmineTest
+import org.scalajs.testsuite.javalib.ExpectExceptions
+
+object ObjectsTest extends JasmineTest with ExpectExceptions {
+
+  describe("java.util.Objects") {
+
+    it("should provide `equals`") {
+      val obj = new Object
+      expect(ju.Objects.equals(obj, obj)).toBeTruthy
+      expect(ju.Objects.equals(new Object, new Object)).toBeFalsy
+      expect(ju.Objects.equals(1, 1)).toBeTruthy
+      expect(ju.Objects.equals(1, 2)).toBeFalsy
+      expect(ju.Objects.equals("abc", "abc")).toBeTruthy
+      expect(ju.Objects.equals("abc", "abd")).toBeFalsy
+    }
```

Comment:
The whole point of `equals` is to deal with `null`s, and you don't test `null`s ;-)
Same for most of the other methods.